recipes/d2l/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "d2l" %}
+{% set version = "0.17.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/d2l-{{ version }}.tar.gz
+  sha256: aee5553431a68246ca315920aaa4d295f6243bf94f8fd0e22b28b9933695d89e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
```

Comment:
Can you add a Github URL as a comment to these tight pins? We normally advise against them but having a link that proves upstream's tight pins would be helpful for future reviewers.